dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -869,6 +909,44 @@ def main() -> None:
         )
         print(f"{tablename} metadata saved to {metadata_root_path}/")
 
+    print("Calculating failure notifications")
+    # Generate a map from SNS topic names -> list of test results that should
+    # be sent to that SNS topic
+    failures_by_topic: typing.Dict[str, typing.List[TestResult]] = {}
+    for test_category in test_categories:
+        for test_result in test_category.test_results:
+            if test_result.status == Status.FAIL and test_result.notify:
+                if test_result.notify not in failures_by_topic:
+                    failures_by_topic[test_result.notify] = []
+                failures_by_topic[test_result.notify].append(test_result)
+
+    # Generate the notification body and send it to each SNS topic. Start
+    # by parsing out a message body and subject for each group of failures
+    # by topic into a list of objects with the keys `topic_arn`, `subject`,
+    # and `body`
+    failure_notifications: typing.List[typing.Dict] = []
+    for topic_arn, test_results in failures_by_topic.items():
+        body = "The following iasWorld data tests are failing:"
+        for test_result in test_results:
+            body += f"\n\n{test_result.details}"
+        failure_notifications.append(
+            {
+                "topic_arn": topic_arn,
+                "subject": "iasWorld data tests failed",
```

Comment:
**suggestion (non-blocking):** Let's maybe add a count to the subject line i.e. `4 iasWorld data tests failed`.

Reply:
Good idea, done in c5096a8aeefab0bb059b8275aaf625a302fb69d0.